modules/oxide/src/less/theme/content/accessibility-checker/accessibility-checker.less

```diff
@@ -38,14 +42,24 @@
 [data-ephox-foam-a11y-violation][data-ephox-foam-a11y-current-violation="info"] {
   outline-color: @accessibility-issue-info-selection-color;
   background-color: @accessibility-issue-info-selection-background-color;
+  box-shadow: 0 0 0 @accessibility-box-shadow-width @accessibility-box-shadow-color;
+  z-index: @accessibility-box-shadow-z-index;
```

Comment:
Wouldn’t it make sense to apply these two styles to the `[data-ephox-foam-a11y-violation][data-ephox-foam-a11y-current-violation]` above (L25) to avoid the unnecessary repetition?

Reply:
no, it will then create x amount of box shadow on the same plain, dragging down performance and block the highlight content

Reply:
I was under the impression that there is only one `data-ephox-foam-a11y-current-violation` at the time so I don’t see the reason why there will be multiple shadows rendered. But anyways, that’s a pretty small nitpick so either way is good.